gen3-integration-tests/conftest.py

```diff
@@ -80,20 +84,81 @@ def pytest_configure(config):
 
 
 @pytest.fixture(autouse=True, scope="session")
+def setup_tests():
+    get_configuration_files()
+    generate_graph_data()
+
+
 def get_configuration_files():
     """
     Get configuration files from the admin VM and save them at `test_data/configuration`
     """
     print("Creating configuration files")
     configs = gen3_admin_tasks.get_admin_vm_configurations(pytest.namespace)
-    path = Path(__file__).parent / "test_data/configuration"
+    path = TEST_DATA_PATH / "configuration"
```

Comment:
Maybe a note that this is a `PosixPath` and not a str object. I was very confused by the slash for what I assumed was string concat

Reply:
yeah i was confused too, it's used everywhere in the existing code. But i'm not sure where i'd put a note. The variable initialization itself is pretty clear

Reply:
maybe renaming to `TEST_DATA_PATH_OBJECT` might be more helpful? Anything to imply it's NOT a string